cobbler/settings/migrations/V3_3_0.py

```diff
@@ -190,7 +214,18 @@ def migrate(settings: dict) -> dict:
                     'bind_zonefile_path': "@@bind_zonefiles@@",
                     'manage_dhcp_v4': False,
                     'manage_dhcp_v6': False,
-                    'next_server_v6': "::1"}
+                    'next_server_v6': "::1",
+                    'bootloaders_formats': {},
+                    'bootloaders_modules': ["btrfs", "ext2", "xfs", "jfs", "reiserfs",
+                                            "all_video", "boot", "cat", "configfile", "echo", "fat", "font", "gfxmenu",
+                                            "gfxterm", "gzio", "halt", "iso9660", "jpeg", "linux", "loadenv", "minicmd",
+                                            "normal", "part_apple", "part_gpt", "part_msdos", "password_pbkdf2", "png",
+                                            "reboot", "search", "search_fs_file", "search_fs_uuid", "search_label",
+                                            "sleep", "test", "true", "video", "mdraid09", "mdraid1x", "lvm", "serial",
+                                            "regexp", "tr", "tftp", "http", "luks", "gcry_rijndael", "gcry_sha1",
+                                            "gcry_sha256"],
+                    'syslinux_dir': "/usr/share/syslinux",
+                    'grub2_mod_dir': "/usr/share/grub2"}
```

Comment:
This could have been place to the other bootloader entries above to keep the alphabetic ordering.

Reply:
Done with: ba1884d